### DISCUSSION TITLE
dependencies error

### DISCUSSION THREAD

**Discussion by joelle78** [vercel/community#5666] [Help] [answered]
### Summary

dependencies error...

### Steps to Reproduce

Can someone help me with this error:

npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @sveltejs/adapter-auto@2.1.0
npm ERR! Found: @sveltejs/kit@2.5.0
npm ERR! node_modules/@sveltejs/kit
npm ERR!   dev @sveltejs/kit@"^2.4.0" from the root project
npm ERR!   peer @sveltejs/kit@"^2.4.0" from @sveltejs/adapter-vercel@5.1.0
npm ERR!   node_modules/@sveltejs/adapter-vercel
npm ERR!     dev @sveltejs/adapter-vercel@"^5.1.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @sveltejs/kit@"^1.0.0" from @sveltejs/adapter-auto@2.1.0
npm ERR! node_modules/@sveltejs/adapter-auto
npm ERR!   dev @sveltejs/adapter-auto@"^2.0.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @sveltejs/kit@1.30.3
npm ERR! node_modules/@sveltejs/kit
npm ERR!   peer @sveltejs/kit@"^1.0.0" from @sveltejs/adapter-auto@2.1.0
npm ERR!   node_modules/@sveltejs/adapter-auto
npm ERR!     dev @sveltejs/adapter-auto@"^2.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /vercel/.npm/_logs/2024-01-29T21_52_25_990Z-eresolve-report.txt
npm ERR! A complete log of this run can be found in: /vercel/.npm/_logs/2024-01-29T21_52_25_990Z-debug-0.log
Error: Command "npm install" exited with 1

My dependencies are looking like this:

 "devDependencies": {
    "@sveltejs/adapter-auto": "^2.0.0",
    "@sveltejs/adapter-vercel": "^5.1.0",
    "@sveltejs/kit": "^2.4.0",
    "svelte": "^4.0.5",
    "vite": "^4.4.2"
  },

Hope someone can help me, i'm not that experienced with deploying via Vercel 😦

**Comment by amyegan** [comment, marked as answer]
Hey @joelle78. I wasn't able to reproduce the error with my SvelteKit test app. But I did notice some package version differences between what you reported and what I have in my fresh SvelteKit starter app.

Try upgrading `@sveltejs/adapter-auto` to `^3.0.0` and make sure your project is using the `SvelteKit (v1)` framework preset in [Build and Development Settings](https://vercel.com/docs/deployments/configure-a-build#build-and-development-settings). And please let me know if that solves it for you 🙏
